src/Microsoft.AspNetCore.DataProtection/DataProtectionUtilityExtensions.cs

```diff
@@ -37,10 +36,6 @@ public static string GetApplicationUniqueIdentifier(this IServiceProvider servic
             if (services != null)
             {
                 discriminator = services.GetService<IApplicationDiscriminator>()?.Discriminator;
```

Comment:
@PinpointTownes as long as it works with null (which it seems like it does) I guess that's fine.